ethereum/contracts/ParachainLightClient.sol

```diff
@@ -36,47 +29,49 @@ library ParachainLightClient {
         bytes32 nextAuthoritySetRoot;
     }
 
+    bytes4 public constant PARACHAIN_ID_SCALE = 0xe8030000;
```

Comment:
Could you rather add the ParaId as constructor param, and then in the constructor use `ScaleCodec.encode32` to get a copy of the encoded bytes. This helps us when the ParaId is no longer 1000.

Reply:
This contract is a library, so has no constructor/state. Going forward I think either we can wrap it in a contract for our specific parachain and put the ParaId there as you say, or put the id into the channel contract, or just deploy a new library for new ParaIds.

Also, I believe using a constant may result in cheaper gas costs not just on the original deployment, but on every future calls into the contract that read that constant, because that constant is stored in the contract code rather than in storage. If so, we should keep it as a constant. I'll double check if this is the case or not.